pkg/lang/ir/system.go

```diff
@@ -50,30 +50,30 @@ func (g Graph) compileRun(root llb.State) llb.State {
 	if len(g.Exec) == 0 {
 		return root
 	}
-	root = root.AddEnv("PATH", types.DefaultPathEnvUnix)
-	logrus.Debugf("compile run: %s", strings.Join(g.Exec, " "))
-	if len(g.Exec) == 1 {
-		return root.Run(llb.Shlex(fmt.Sprintf("bash -c \"%s\"", g.Exec[0]))).Root()
-	}
-
-	var sb strings.Builder
-	sb.WriteString("set -euo pipefail\n")
-	for _, c := range g.Exec {
-		sb.WriteString(c + "\n")
-	}
 
-	cmdStr := fmt.Sprintf("bash -c '%s'", sb.String())
-	logrus.WithField("command", cmdStr).Debug("compile run command")
 	workingDir := g.getWorkingDir()
-	run := root.Dir(workingDir).
-		Run(llb.Shlex(cmdStr))
-	// Mount the build context into the build process.
-	// TODO(gaocegege): Maybe we should make it readonly,
-	// but these cases then cannot be supported:
-	// run(commands=["git clone xx.git"])
-	run.AddMount(workingDir, llb.Local(flag.FlagBuildContext))
+	stage := root.AddEnv("PATH", types.DefaultPathEnvUnix)
```

Comment:
Do we need this? We already get https://github.com/tensorchord/envd/pull/1044/files#diff-92a46cac85058f5f71561e979e5496306759771d870bc6b46c9ec2c30c4e5c68R68

Reply:
removed